dali/kernels/CMakeLists.txt

```diff
@@ -14,10 +14,11 @@
 
 project(kernels)
 
-add_subdirectory(test)
+add_subdirectory(algebra)
```

Comment:
I wouldn't be looking for such an operator in "algebra". It's a pointwise operation and I'd look for something like "pixel", "pointwise" or "imgproc", whereas "algebra" is something that suggests operations on linear systems - like multiplying large matrices, solving linear systems or performing tensor contractions.

Reply:
Done